client/components/jetpack/connection-health/index.tsx

```diff
@@ -79,6 +81,34 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		);
 	}
 
+	if ( errorType === REST_API_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate( 'Jetpack encounters REST API connection issues.' ) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-an-issue-with-your-sites-jetpack-connection'
```

Comment:
I created an issue to add an explanation to Resolve Jetpack Errors: https://github.com/Automattic/en.support-docs-content/issues/2080

Reply:
Should I create a ticket to revisit this and update the URL after the new docs are in place?